katalog/configs/kubernetes/cluster-flow.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: ClusterFlow
+metadata:
+  name: kubernetes
+spec:
+  filters:
+    - dedot:
+        de_dot_separator: "_"
+        de_dot_nested: true
+  match:
+    - exclude:
+        namespaces:
+          - ingress-nginx
```

Comment:
I'm guessing that we put this exclusion because we have already the `ingress-nginx` specific config, but, if I understand correctly, the `nginx-ingress` config flow is only for the ingress controller pods (actually the pods with that math the label selector).

What happens if we have other stuff on the namespace that doesn't have that label (`app: ingress-nginx`)? forecastle, for example.

Reply:
Improved, now the clusterflow is only skipping ingress nginx logs